src/finality/justification/decode.rs

```diff
@@ -176,7 +248,7 @@ impl<'a> ExactSizeIterator for VotesAncestriesIter<'a> {}
 pub struct Error(nom::error::ErrorKind);
 
 /// Nom combinator that parses a justification.
-fn justification(bytes: &[u8]) -> nom::IResult<&[u8], JustificationRef> {
+pub fn justification(bytes: &[u8]) -> nom::IResult<&[u8], JustificationRef> {
```

Comment:
Shouldn't be public. Please create a function `decode_partial` similar to `decode`.